lib/brainstem/query_strategies/filter_and_search.rb

```diff
@@ -7,10 +7,9 @@ def execute(scope)
 
         if ordering?
           count_scope = scope
-          scope = paginate(scope)
+          scope = paginated_scope(scope)
           scope = @options[:primary_presenter].apply_ordering_to_scope(scope, @options[:params])
-          primary_models = evaluate_scope(scope)
-          count = evaluate_count(count_scope)
+          primary_models, count = paginator.paginate(page: calculate_page, scope: scope, count_scope: count_scope)
```

Comment:
So we `paginate` twice? It seems like the limit and offset that we're doing in FilterAndSearch should be a responsibility of the Paginator.